### PR TITLE
fixed README, added .gitignore, added args parser

### PR DESCRIPTION
**Pull request by Mistra**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.pyc
```
Comment (nolash) [inline .gitignore]: what's this red character?

File: scripts/python/singlepss.py [hunk 2]
```diff
@@ -61,9 +61,9 @@
 # stream processor
 stream = pss.Stream()
 
-
 # all weechat scripts must run this as first function
-weechat.register("pss", "lash", PSS_VERSION, "GPLv3", "single-node pss chat", "pss_stop", "")
+weechat.register("pss", "lash", PSS_VERSION, "GPLv3", "single-node pss chat",
+                 "pss_stop", "")
```
Comment (nolash) [inline scripts/python/singlepss.py]: either all on separate tabs or none?

File: scripts/python/singlepss.py [hunk 3]
```diff
@@ -112,11 +112,16 @@ def processFeedBox(pssName, _):
 
 		try:
 			hsh = bzzs[pssName].add(update.data)
-			wOut(PSS_BUFPFX_DEBUG, [], ">>>", "bzz sent for " + pssName + "." + update.name + ": " + hsh)
+			wOut(PSS_BUFPFX_DEBUG, [], ">>>",
+			     "bzz sent for " + pssName + "." + update.name + ": " + hsh)
```
Comment (nolash) [inline scripts/python/singlepss.py]: same here, etc...

File: scripts/python/singlepss.py [hunk 4]
```diff
@@ -203,9 +209,15 @@ def buf_get(pssName, typ, name, known):
 			shortname = "pss:" + name[:8]
 			ispubkey = True
 
-		buf = weechat.buffer_new(bufname, "buf_in", pssName, "buf_close", pssName)
+		buf = weechat.buffer_new(bufname, "buf_in", pssName, "buf_close",
+		                         pssName)
 		weechat.buffer_set(buf, "short_name", shortname)
-		weechat.buffer_set(buf, "title", name + " @ PSS '" + pssName + "' | node: " + weechat.config_get_plugin(psses[pssName].host + "_url") + ":" + weechat.config_get_plugin(psses[pssName].port + "_port") + " | key  " + pss.label(psses[pssName].key) + " | address " + pss.label(psses[pssName].base))
+		weechat.buffer_set(
+		    buf, "title", name + " @ PSS '" + pssName + "' | node: " +
```
Comment (nolash) [inline scripts/python/singlepss.py]: yes this is messy. In these cases maybe build long string separately then put in call.

File: scripts/python/singlepss.py [hunk 5]
```diff
@@ -217,10 +229,15 @@ def buf_get(pssName, typ, name, known):
 			else:
 				pubkey = remotekeys[name].decode("hex")
 			try:
-				feeds[name] = pss.Feed(bzzs[pssName].agent, psses[pssName].get_account(), "d" + pss.publickey_to_account(pubkey))
-				wOut(PSS_BUFPFX_DEBUG, [], "", "added feed with topic " + feeds[name].topic.encode("hex"))
+				feeds[name] = pss.Feed(bzzs[pssName].agent,
+				                       psses[pssName].get_account(),
+				                       "d" + pss.publickey_to_account(pubkey))
```
Comment (nolash) [inline scripts/python/singlepss.py]: this looks bizarre in this diff display. Look at the "d" - why so far away? How is this clearer? :)

File: scripts/python/singlepss.py [hunk 6]
```diff
@@ -492,113 +532,118 @@ def buf_node_in(pssName, buf, args):
...
+			     "invalid message " + msg)
 			return weechat.WEECHAT_RC_ERROR
 
 		return buf_in(pssName, buf, msg)
 
 	# output node key
 	elif argv[0] == "key" or argv[0] == "pubkey":
-		wOut(PSS_BUFPFX_INFO, [bufs[pssName]], pssName + ".key", currentPss.key)
-
+		wOut(PSS_BUFPFX_INFO, [bufs[pssName]], pssName + ".key",
```
Comment (nolash) [inline scripts/python/singlepss.py]: while I'm reading this, I'm even not sure if splitting these into multiple lines will make them clearer. In the end this is only output to client, and the `wOut` function prefix is very distinguishable. Maybe just revert this.

File: scripts/python/singlepss.py [hunk 7]
```diff
@@ -608,80 +653,75 @@ def pss_sighandler_load(data, sig, sigdata):
...
 	except IOError as e:
-		wOut(PSS_BUFPFX_WARN, [], "!!!", "could not open contact store " + scriptPath + "/.pss-contacts: " + repr(e))
-	
+		wOut(
+		    PSS_BUFPFX_WARN, [], "!!!", "could not open contact store " +
+		    scriptPath + "/.pss-contacts: " + repr(e))
+
 	# open file for append (and create if not exist)
 	# \todo postfix db name with username
 	storeFile = open(scriptPath + "/.pss-contacts", "a", 0600)
```
Comment (nolash) [inline scripts/python/singlepss.py]: What does this red highlight mean?

File: scripts/python/singlepss.py [hunk 8]
```diff
@@ -608,80 +653,75 @@ def pss_sighandler_load(data, sig, sigdata):
...
-	" <name>     add: add new contact\n"
-	" <name>     msg: send message to contact\n"
-	" <name>    stop: disconnect from node\n"
-	" <name>     key: display public key\n"
-	" <name>    addr: display overlay address\n"
-	"\n",
-	"(if this displays anywhere please tell me)",
-	"pss_handle",
-	"")
+    "pss", "connect to pss node and manage local contact list",
```
Comment (nolash) [inline scripts/python/singlepss.py]: No one per line please. We can build the multiline help string above, that may make it clearer.

File: scripts/python/singlepss.py (hunk 7 above)
Comment (Mistra) [reply scripts/python/singlepss.py]: Apparently it's github.

File: .gitignore (hunk 1 above)
Comment (Mistra) [reply .gitignore]: I think it's github, for some reason it complains because there's no newline. Adding one.

**Pull request closed without merging**